lib/git-lfs-s3/services/upload/upload_required.rb

```diff
@@ -26,7 +28,11 @@ def status
       private
 
       def upload_destination
-        object.presigned_url(:put)
+        if ceph_s3
+          GitLfsS3::CephPresignerService::signed_url(object)
+        else
+          object.presigned_url(:put)
```

Comment:
I've tested ` object.presigned_url(:put, acl: 'public-read')` as a replacement for this line and objects are created with the correct permission under s3.